### ISSUE TITLE
A11y_Visual Studio Code Services_Docs Python_Tutorial_DeveloperTool: Incorrect role given for "Edit" control as "link" and "Button"

### ISSUE THREAD

**Issue by prabhudevu** [microsoft/vscode-docs#5554]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags**:#A11yMAS;#A11yTCS;#BM_Visual Studio Code Services_Aug2022;#Visual Studio Code Services;#Win11;#FTP;#DesktopWeb;#A11ySev2;#Chrome;#WCAG4.1.2;#DeveloperTool;

## Environment Details:
Application Name: Visual Studio Code Services
URL: https://code.visualstudio.com/
Windows Version: Win11
Chrome Version: Version 103.0.5060.134 (Official Build) (64-bit)
Screen reader: NVDA (version 2022.1), JAWS (version 2022.2112.24 ILM)

## Repro Steps:

1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix
2. Launch the application Url : https://code.visualstudio.com/
3. Tab Navigate to "Docs" link.
4. Tab Navigate to "Tutorial" link under "Python" in left Navigation and hit enter
5. Turn on the screen reader 
6. Tab Navigate to "Edit" beside "Getting Started with Python in VS Code" heading
7. Observe that Screen reader is announcing both "link" and "button" when focus is on "Edit" .Role of "Edit" is given as both link and button

## Actual:

Role of "Edit" is given as both link and button, Screen reader is announcing both "link" and "button" when focus is on "Edit" control on using tab key
**Note:** 

1. Issue is observed throughout the application
2. Issue is Observed with JAWS screen reader as well

## Expected:

Role of "Edit" should be only given as "link" button role should be removed. On activating "Edit" control the URL is getting changed so the role should only be "link"

## User Impact:
Screen reader users will find it difficult to know the role if Screen reader is announcing both "link" and "button" when focus is on "Edit" control

## Attachment
![5554_A11y_Visual Studio Code Services_Docs Python_Tutorial_DeveloperTool](https://user-images.githubusercontent.com/98821792/183332029-2a27363e-c111-45d6-b625-5cbbba3779ae.gif)
![5554_A11y_Visual Studio Code Services_Docs Python_Tutorial_DeveloperTool](https://user-images.githubusercontent.com/98821792/185074082-8857314d-cb00-4e0a-a61c-615f6f178cd5.png)

**Comment by prabhudevu** [comment]
![5554_closed](https://user-images.githubusercontent.com/98821792/216281445-4fc162cb-1b01-40cd-be99-9e28f66c90e3.png)

I have verified the issue in the Env: https://code.visualstudio.com/docs/python/python-tutorial
Issue is fixed, hence closing the bug. 

GithubTags: #Closed;
